test/recipes/80-test_ssl.t

```diff
@@ -603,10 +603,28 @@ sub testssl {
 	}
     };
 
+    subtest 'SNI tests' => sub {
+
+	plan tests => 7;
+
+      SKIP: {
+	  skip "TLSv1.0 is not supported by this OpenSSL build", 7
```

Comment:
Why are these run with TLS 1.0 rather than the default, version-flexible method?